mipengine/node/monetdb_interface/views.py

```diff
@@ -13,15 +14,23 @@ def get_view_names(context_id: str) -> List[str]:
 
 @validate_identifier_names
 def create_view(
-    view_name: str, pathology: str, datasets: List[str], columns: List[str]
+    view_name: str,
+    pathology: str,
+    datasets: List[str],
+    columns: List[str],
+    filters: str,
 ):
-    # TODO: Add filters argument
+    filter_clause = ""
+    if filters is not None:
+        validate_proper_filter(pathology, filters)
```

Comment:
As we have discussed, we need to move this piece from the `create_view` to the controller api.

The validation will be made on the controller and on the node, only an sql_injection check.